solutions/project/spoke-project-unclass/network/dns.yaml

```diff
@@ -0,0 +1,150 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Enable DNS logging
+apiVersion: dns.cnrm.cloud.google.com/v1beta1
+kind: DNSPolicy
+metadata:
+  name: project-id-logging-dnspolicy # kpt-set: ${project-id}-logging-dnspolicy
+  namespace: networking
+  annotations:
+    cnrm.cloud.google.com/project-id: project-id # kpt-set: ${project-id}
+    config.kubernetes.io/depends-on: compute.cnrm.cloud.google.com/namespaces/networking/ComputeNetwork/project-id-global-vpc1-vpc # kpt-set: compute.cnrm.cloud.google.com/namespaces/networking/ComputeNetwork/${project-id}-global-vpc1-vpc
+spec:
+  resourceID: logging-dnspolicy
+  description: "DNS policy to enable logging"
+  enableLogging: true
+  networks:
+    - networkRef:
+        name: project-id-global-vpc1-vpc # kpt-set: ${project-id}-global-vpc1-vpc
+---
+# Private dns zone for googleapis.com
+apiVersion: dns.cnrm.cloud.google.com/v1beta1
+kind: DNSManagedZone
+metadata:
+  name: googleapis
```

Comment:
metadata/name needs to be unique inside a namespace. considering this spoke package will be deployed several times in a landing zone, all resource name should follow this example : `name: project-id-host-vpc # kpt-set: ${project-id}-host-vpc`

Reply:
follow this standard for every resources in this package

Reply:
done

Reply:
@uday-ssc  also try to use the naming convention document to define the suffix (example "-vpc") that should be added at the end of the resource name. See section 1.6 of this [doc](https://163gc.sharepoint.com/:w:/r/sites/SSCSPC-CTODPT-GCP/Shared%20Documents/Documentation/External%20Documentation/Supporting%20Document/GCP%20Naming%20and%20Tagging/SSC%20GCP%20Naming%20and%20Tagging%20Quick%20Reference%20v0.1.docx?d=wb164736f8954473ab8a9065acaf65f1e&csf=1&web=1&e=FnazDT)